File: Selenium/Activity-12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait
with webdriver.Firefox() as driver:
   wait = WebDriverWait(driver, timeout=10)
   driver.get("https://training-support.net/webelements/dynamic-content")
   print(driver.title)
   driver.find_element(By.XPATH,"//button[text()='Click me!']").click()

   if(wait.until(EC.text_to_be_present_in_element((By.ID,"word"),"release"))):
        print(driver.find_element(By.ID,"word").text)
driver.quit()

File: Selenium/Activities/Activity_12.java
import java.time.Duration;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity_12 {
	public static void main(String[] args) {
		WebDriver driver=new FirefoxDriver();
		driver.get("https://training-support.net/webelements/dynamic-content");
		System.out.println(driver.getTitle());
		WebDriverWait wait=new WebDriverWait(driver, Duration.ofSeconds(10));
		
		WebElement Click=driver.findElement(By.xpath("//button[text()='Click me!']"));
		Click.click();
		
		if(wait.until(ExpectedConditions.textToBePresentInElementLocated(By.id("word"),"release"))) 
			System.out.println(driver.findElement(By.id("word")).getText());	
		driver.quit();	
}
}